stacks/checkbrackets.test.js

```diff
@@ -0,0 +1,43 @@
+const checkBrackets = require('./checkBrackets');
+
+test('test if the brackets are balanced, each opening bracket has one closing bracket then checkBrackets() should return true', () => {
+    var testString = "sqrt(5*(3+8)/ (4-0) )";
+    var testStack = checkBrackets(testString);
+    expect(testStack).toBeTruthy();
+});
+
+test('test if the brackets are not balanced then checkBrackets() should return false', () => {
```

Comment:
two tests can not have same name line 9 and line 15

Reply:
removed duplicate one